src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -393,6 +421,11 @@ - (void)setFrame:(CGRect)frame {
   [self layoutVisiblePages];
 }
 
+- (void)layoutSubviews {
+  _scrollView.contentSize = [self contentSizeForPagingScrollView];
+  [self layoutVisiblePages];
```

Comment:
Why do we need to do this now?

You should also _always_ call `[super layoutSubviews]`; Not doing so can break things like autolayout constraints (and so also autoresizing masks).